.github/workflows/code_validation.yml

```diff
@@ -1,45 +1,66 @@
-name: Linting
+name: Code validation
 
 on:
   push:
     branches: [main]
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+*
```

Comment:
Do we really want to run code validation when tagging a release? It should have been run (and fixed) before you do a tag?

Reply:
It's not explicitly wrong, just feels kind of weird?

Reply:
I think you're right, it doesn't make much sense to run validation on code that you are in some capacity saying is already valid.